initcpio/hooks/oroot

```diff
@@ -39,7 +38,7 @@ run_hook() {
               mount -o \$(awk '{if (\$2 == \"/\") {print \$4}}' /etc/fstab) \"$root\" /run/oroot || mount \"$root\" /run/oroot;
               rsync -ax --no-whole-file --inplace --delete / /run/oroot --exclude boot --exclude dev --exclude mnt --exclude proc --exclude run --exclude sys --exclude tmp --exclude usr/bin/overlay_flush;
               umount /run/oroot &&
-              rm -r /run/oroot &&
+              rmdir /run/oroot &&
```

Comment:
Stupid github put the comment on the wrong line, there's one case in the file where there is no cryptsetup close